cranelift/codegen/src/isa/aarch64/lower.rs

```diff
@@ -550,13 +550,13 @@ fn collect_address_addends(
                         result32.push((reg, extendop));
                     }
                 }
-                Opcode::Uextend | Opcode::Sextend => {
```

Comment:
Did you mean to remove the `Sextend` case?

Reply:
Argh, no, I was playing with refactoring this a bit (splitting by uextend/sextend toplevel) and didn't revert completely. Thanks for catching!

Reply:
Also, now that I'm looking at this again, I had a momentary panic that we had another sign-extend bug (déjà-vu!); the reason that this *isn't*, for anyone watching, is that for narrower-than-32 extends we push the *extended value* (the output of the extend op) onto `addends64`, rather than the input to the extend. The 32-bit case is handled above specially with the extend incorporated into the amode if possible.